src/zed_host/rviz/zed_preview.rviz

```diff
@@ -335,6 +335,16 @@ Visualization Manager:
         {}
       Queue Size: 100
       Value: true
+    - Alpha: 0.699999988079071
+      Class: rviz/Map
+      Color Scheme: map
+      Draw Behind: false
+      Enabled: true
+      Name: Cone "Walls"
+      Topic: /map
+      Unreliable: false
+      Use Timestamp: false
+      Value: true
```

Comment:
I thought zed stuff shouldn't be touched. Was a test executed that modified this?

Reply:
The `/map` topic wasn't being shown in RViz until this change. This change doesn't have much to do with zed, other than the RViz config file being stored in its package.